cmd/mesh/rpc_handler.go

```diff
@@ -112,9 +112,6 @@ func SetupOrderStream(ctx context.Context, app *core.App) (*ethRpc.Subscription,
 
 		for {
 			select {
-			case <-ctx.Done():
```

Comment:
This fixes a bug in #304 that was previously unnoticed. `ctx` here is not something we control and it seems to be canceled even though a client is still actively subscribed. I can't find any documentation about the expected behavior of `ctx` at https://godoc.org/github.com/ethereum/go-ethereum/rpc. For now, we have no real choice but to ignore the given context.